certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -349,6 +349,116 @@
+          "x-ms-dynamic-schema": {
+            "operationId": "CPGetActionOutputSchema",
+            "parameters": {
+              "action_ref": {
+                "parameter": "action"
+              }
+            },
+            "value-path": "output"
+          },
+          "x-ms-dynamic-properties": {
+            "operationId": "CPGetActionOutputSchema",
+            "parameters": {
+              "action_ref": {
+                "parameterReference": "actionReq/action"
+              }
+            },
+            "itemValuePath": "output"
+          }
+        }
+      },
+      "post": {
+        "description": "CyberProof User Action.",
+        "summary": "CyberProof User Action",
+        "operationId": "CPCreateWebhookExecution",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "actionReq",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required": [
+                "action",
+                "parameters",
+                "url"
+              ],
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "The callback URL.",
+                  "x-ms-url-encoding": "single",
+                  "x-ms-summary": "Url",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true,
+                  "title": "Url"
+                },
+                "action": {
+                  "type": "string",
+                  "description": "select action",
+                  "x-ms-url-encoding": "single",
+                  "x-ms-summary": "Select Action",
+                  "x-ms-dynamic-values": {
+                    "operationId": "CPGetWebhooksActions",
+                    "value-path": "ref",
+                    "value-title": "path"
+                  }
+                },
+                "parameters": {
+                  "description": "parameters description",
+                  "type": "object",
+                  "x-ms-dynamic-schema": {
+                    "operationId": "CPGetActionSchema",
+                    "parameters": {
+                      "action_ref": {
+                        "parameter": "action"
+                      }
+                    },
+                    "value-path": "items"
+                  },
+                  "x-ms-dynamic-properties": {
+                    "operationId": "CPGetActionSchema",
+                    "parameters": {
+                      "action_ref": {
+                        "parameterReference": "actionReq/action"
+                      }
+                    },
+                    "itemValuePath": "items"
+                  }
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Webhook user action was triggered successfully",
+            "schema": {}
```

Comment:
@Avibek Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

Reply:
@sriyen-msft But this is internal operation.
So makers won't be exposed to it.
We discussed this in our [last PR](https://github.com/microsoft/PowerPlatformConnectors/pull/1617#discussion_r867969814)